dotnet/WPF/VisualLayerIntegration/VisualLayerIntegration/CompositionHost.cs

```diff
@@ -220,42 +171,21 @@ internal struct DispatcherQueueOptions
             public DISPATCHERQUEUE_THREAD_APARTMENTTYPE apartmentType;
         };
 
-        // Win32 method signature we're duplicating.
-        //HRESULT CreateDispatcherQueueController(
-        //  DispatcherQueueOptions options,
-        //  ABI::Windows::System::IDispatcherQueueController** dispatcherQueueController
-        //);
         [DllImport("coremessaging.dll", EntryPoint = "CreateDispatcherQueueController", CharSet = CharSet.Unicode)]
```

Comment:
Simplify the attribute - no need to name the entrypoint if it has the same name as your method (which it seems to here) and we should be able to not set the CharSet here because no strings are involved.

Reply:
updated